test/lit.cfg

```diff
@@ -1500,7 +1500,8 @@ else:
         config.available_features.add('libdispatch')
     else:
         # TSan runtime requires libdispatch on non-Apple platforms
-        config.available_features.remove('tsan_runtime')
+        if 'tsan_runtime' in config.available_features:
```

Comment:
We can use `discard` instead of `remove` (does not throw if not present).

Reply:
Good idea, I'll update this.